src/admin-views/aggregator/origins/ics.php

```diff
@@ -44,6 +44,11 @@ class="tribe-ea-field tribe-ea-media_button tribe-dependent button button-second
 			esc_html_e( 'Preview', 'the-events-calendar' );
 			?>
 		</button>
+		<button type="button" class="button tribe-clear-filters">
+			<?php
+			esc_html_e( 'Clear Filters', 'the-events-calendar' );
```

Comment:
Totally get that this is in keeping with the style we're using elsewhere in this section, but why not a one liner, PHP tags and all? Again, not a big deal.